rllib/examples/mobilenet_v2_with_lstm.py

```diff
@@ -43,12 +47,13 @@
     # Configure our Trainer.
     config = {
         "env": RandomEnv,
-        "framework": "torch" if args.torch else "tf",
+        "framework": args.framework,
         "model": {
             "custom_model": "my_model",
             # Extra config passed to the custom model's c'tor as kwargs.
             "custom_model_config": {
-                "cnn_shape": cnn_shape_torch if args.torch else cnn_shape,
+                "cnn_shape": cnn_shape_torch
```

Comment:
Add comment to explain that the channel dimension is permuted in Torch's case.

Reply:
done